manifests/sentinel.pp

```diff
@@ -149,6 +149,8 @@
     }
   }
 
+  $sentinel_bind_arr = [$sentinel_bind].flatten
```

Comment:
see delete_undef_values for @ekohl's comment, or it's documentation for how to do that with filter